bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/internal/findandreplace/overlay/FindReplaceOverlay.java

```diff
@@ -303,6 +303,11 @@ private void focusTargetWidget() {
 			targetWidget.forceFocus();
 		}
 	};
+	private KeyListener closeOnEditorEscapeListener = KeyListener.keyPressedAdapter(c -> {
```

Comment:
very nitpicky: you could generalize this to `closeOnTargetWidgetEscapeListener` or the like, since it is (at least here and in the future hopefully everywhere in the code) not necessarily an editor.

Reply:
I agree with you, this behavior should still be true when targetting the outline for example! Addressed in https://github.com/eclipse-platform/eclipse.platform.ui/commit/03eaebeb9785bc814ade730a274c6e6708de6c57